src/SFA.DAS.Assessor.Functions/Functions/ExternalApiDataSync/ExternalApiDataSyncFunction.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using SFA.DAS.Assessor.Functions.ExternalApis.Assessor;
+using SFA.DAS.Assessor.Functions.ExternalApis.Assessor.Types;
+
+namespace SFA.DAS.Assessor.Functions.Functions.ExternalApiDataSync
```

Comment:
RebuildExternalApiSandboxFunction?? - seems like a better name - and you are free to rename it here - just make everything else follow along